src/cache.js

```diff
@@ -2,18 +2,18 @@
 
 var _cache = [];
 
-function EventHandler(token, eventName, handler, enabled) {
+function EventHandler(token, eventName, splitEventName, handler, enabled) {
```

Comment:
I'd be inclined to pass the descriptor here instead of the eventName and the splitEventName, Then split the descriptor to assign props to the eventHandler in the constructor.